### PR TITLE
[HIPIFY] Update issue_report.yml to support MI300 and ROCm v6.0.0

### PR DESCRIPTION
**Pull request by abhimeda**
Added MI300, ROCm v6, and set a default component

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_report.yml [hunk 1]
```diff
@@ -39,121 +39,124 @@ body:
     placeholder: "e.g. AMD Ryzen 9 5900HX with Radeon Graphics"
   validations:
     required: true
+- type: dropdown
+  attributes:
+    label: GPU
```
Comment (emankov) [inline .github/ISSUE_TEMPLATE/issue_report.yml]: I believe in #1191 we agreed to exclude the GPU field.

File: .github/ISSUE_TEMPLATE/issue_report.yml [hunk 2]
```diff
@@ -39,121 +39,124 @@ body:
...
     options:
+        - ROCm 6.0.0
         - ROCm 5.7.1
         - ROCm 5.7.0
         - ROCm 5.6.0
         - ROCm 5.5.1
         - ROCm 5.5.0
   validations:
-    required: false
+    required: true
```
Comment (emankov) [inline .github/ISSUE_TEMPLATE/issue_report.yml]: not required

**Review by emankov — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/issue_report.yml (hunk 2 above)
Comment (abhimeda) [reply .github/ISSUE_TEMPLATE/issue_report.yml]: removed the gpu section

Comment (emankov) [reply .github/ISSUE_TEMPLATE/issue_report.yml]: required: false

Comment (abhimeda) [reply .github/ISSUE_TEMPLATE/issue_report.yml]: done!

**Review by emankov — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**